#169392007 fix ui issues

Ntare22 opened the pull request:
> #### What does this PR do?
* Fixes responsive issues on the website
> #### Description of Task to be completed
* create page to view single entry
* create page to update an entry
* create hamburger button for a responsive navbar
* create humburger icon (HTML & CSS)
* create dropdown menu
* fix responsiveness issues.
* delete commented out code
> #### How should this be manually tested?
* clone this repo
* enter ui folder
* click on index.html
* check for responsiveness.
> #### Any background context you want to provide?
* N/A
> #### What are the relevant pivotal tracker stories?
* [#169392007](https://www.pivotaltracker.com/story/show/169392007)
> #### Screenshots (if appropriate)
![Screen Shot 2019-10-28 at 15 27 55](https://user-images.githubusercontent.com/30793016/67682381-e93d5700-f997-11e9-9a09-fd38c689853b.png)

Commit:
feat(create hamburger button):

create humburger icon (HTML & CSS)
create dropdown menu

[ Finishes #169392007 ]

NkFab commented:
The dropdown menu background should fill the whole screen width or at least fill a part of the screen starting from the position of the hamburger button. Can you please add all the feedback we talked about in this one branch?

Commit:
bg(fix ui issues):

create page to view single entry
create page to update an entry
create hamburger button for a responsive navbar

[ Finishes #169392007 ]

NkFab commented:
Fix this responsiveness issue
![image](https://user-images.githubusercontent.com/22955494/67767806-e73bcc80-fa59-11e9-859a-84f0b10904bc.png)

NkFab commented:
Align the title form to the field where you write your story.
![image](https://user-images.githubusercontent.com/22955494/67767867-08042200-fa5a-11e9-884f-fb4a42f90a78.png)

NkFab commented:
There is no need to still have these check-boxes on the entries since you can see the edit and delete button options while viewing a story. 
![image](https://user-images.githubusercontent.com/22955494/67767950-3255df80-fa5a-11e9-8e09-9bf0550037b9.png)

NkFab commented on ui/css/style.css:
```diff
@@ -114,6 +114,56 @@ div.hamburger {
+/* .dropdown:hover .hamburger-dropdown {
+    display: block;
+    right: -70px;
+} */
```
Remove commented lines.

NkFab commented on ui/css/style.css:
```diff
@@ -610,6 +680,27 @@ footer {
+    position: absolute;
+    background-color: #020024;
+    /* min-width: 160px; */
+    /* min-height: 160px; */
```
Remove commented lines.

NkFab commented on ui/css/style.css:
```diff
@@ -638,6 +729,23 @@ footer {
+    position: absolute;
+    background-color: #020024;
+    /* min-width: 160px; */
+    /* min-height: 160px; */
```
Remove commented lines.

NkFab commented on ui/css/style.css:
```diff
@@ -653,6 +761,23 @@ footer {
+    position: absolute;
+    background-color: #020024;
+    /* min-width: 160px; */
+    /* min-height: 160px; */
```
Remove commented lines.

NkFab commented on ui/css/style.css:
```diff
@@ -674,6 +799,16 @@ footer {
+    position: absolute;
+    background-color: #020024;
+    /* min-width: 160px; */
+    /* min-height: 160px; */
```
Remove commented lines.

NkFab commented on ui/js/index.js:
```diff
@@ -3,15 +3,38 @@
-const popUpBtn = document.getElementById("new-entry");
+
+
+// const updateBtn = document.getElementById("edit-entry");
```
Remove commented lines.

NkFab commented on ui/single-entry.html:
```diff
@@ -50,18 +51,63 @@ <h1>MyDiary</h1>
+      <a class="thoughts-btn save-btn" type="submit" onclick="saveEntry()">Save</a>
     </div>
-  </div>
+  </div> -->
```
Remove commented lines.

Commit:
bg(fix ui issues):

fix responsiveness issues.
delete commented out code

[ Finishes #169392007 ]
